e2e/scripts/setup_e2e.sh

```diff
@@ -54,23 +67,36 @@ CHAOS_NAMESPACE=${CHAOS_NAMESPACE:-"chaos-testing"}
 # Create the Kind cluster with the specified Kubernetes version.
 CLUSTER=${CLUSTER} KUBE_VERSION=${KUBE_VERSION} ${SCRIPT_DIR}/start_kind_cluster.sh
 
-echo "Preload FoundationDB images for version ${FDB_VERSION}"
-fdb_image=$(get_image_name "foundationdb/foundationdb:${FDB_VERSION}")
-fdb_sidecar_image=$(get_image_name "foundationdb/foundationdb-kubernetes-sidecar:${FDB_VERSION}-1")
-docker pull "${fdb_image}"
-docker pull "${fdb_sidecar_image}"
+# Make sure all required versions for the tests are available in the kind cluster.
+preload_foundationdb_images_for_version "${FDB_VERSION}"
+preload_foundationdb_images_for_version "${PREVIOUS_FDB_VERSION}"
 
-preload_foundationdb_images "${CLUSTER}" "${fdb_image}" "${fdb_sidecar_image}"
+for version in $(echo ${UPGRADE_VERSIONS} | tr ':,' ' ');
+do
+  preload_foundationdb_images_for_version "${version}"
+done
 
 echo "Build the operator image from the current revision"
-BUILD_PLATFORM="linux/amd64" make -C "${SCRIPT_DIR}/../.." container-build
-kind load docker-image "fdb-kubernetes-operator:latest" --name "${CLUSTER}"
+operator_image=$(get_image_name "fdb-kubernetes-operator:latest")
+IMG="${operator_image}" BUILD_PLATFORM="linux/amd64" make -C "${SCRIPT_DIR}/../.." container-build
+kind load docker-image "${operator_image}" --name "${CLUSTER}"
 
 echo "Install the CRDs in the Kind cluster"
 kubectl apply -f "${SCRIPT_DIR}/../../config/crd/bases/"
 
 echo "Install the chaos-mesh in the Kind cluster"
 kubectl create ns "${CHAOS_NAMESPACE}" || true
+
+# If helm is not installed install it.
+if ! which helm &> /dev/null;
```

Comment:
We added helm to the build container, so we can drop this soon: https://github.com/FoundationDB/fdb-build-support/pull/66.